spras/omicsintegrator2.py

```diff
@@ -119,12 +119,27 @@ def run(edges=None, prizes=None, output_file=None, w=None, b=None, g=None, noise
             command.extend(['--seed', str(seed)])
 
         container_suffix = "omics-integrator-2:v2"
-        run_container_and_log('Omics Integrator 2',
-                             container_framework,
-                             container_suffix,
-                             command,
-                             volumes,
-                             work_dir)
+
+        # We use this later either if we encounter unrecoverable OI2 errors
+        # or as the main output file we want to post-process in parse_output.
+        output_tsv = Path(out_dir, 'oi2.tsv')
+
+        try:
+            run_container_and_log('Omics Integrator 2',
+                                container_framework,
+                                container_suffix,
+                                command,
+                                volumes,
+                                work_dir)
+        except ContainerError as err:
+            needle = "all the input arrays must have same number of dimensions, but the array at index 0 has 2 dimension(s) and the array at index 1 has 1 dimension(s)"
```

Comment:
@ntalluri I remember you having experience with OI2 errors - I wanted to check if this error only occurs when OI2 gets passed an empty DF, or if I should do a quick-check which makes sure that the DF is empty before ignoring this error.

Reply:
For the empty df, is the error because of empty sources/targets and prizes? I was checking a piece of code in dataset.py and  lines [99-101] have a check for an empty network. 

```
else:
            raise ValueError(
                f"Edge file {interactome_loc} must have three or four columns but found {num_cols}"
            )
```

I would assume we want a similar thing if there are 0 sources/targets or prizes.

I think this PR is not the right place to update this issue though.